run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh --
+
+# Debugging: -x to enable, +x to disable
+set +x
+# uncomment the below line after putting correct information
+#LOG_CFG=/path/to/logging.yaml python groupsforsections.py /path/to/config.yaml
```

Comment:
Set default values for the log and application configuration files.  Test those values and give an error if they are not valid.  The way this shell script is now, it appears to be quietly successful, when it actually does nothing.